utils/oc.py

```diff
@@ -423,14 +426,14 @@ def _run(self, cmd, **kwargs):
                 raise MetaDataAnnotationsTooLongApplyError(
                     f"[{self.server}]: {err}")
             if not (allow_not_found and 'NotFound' in err):
-                raise StatusCodeError(f"[{self.server}]: {err}")
+                logging.error(StatusCodeError(f"[{self.server}]: {err}"))
```

Comment:
This is more conceptual, but when the function raises an exception, the function is already telling all it should tell. Logging the exception message is a decision for the caller, if it wants to do it as part of the exception handling.